README.md

```diff
@@ -1,28 +1,136 @@
-# backend
-
-```python
-python3 -m venv env
-source env/bin/activate
-```
-
-# Start python
-
-```python
-python manage.py runserver
-
-python manage.py migrate
-```
-
-```python
-python manage.py createsuperuser
-
-pip install Django
-```
-
-```python
-pip freeze > requirements.txt
-```
-
-```update
-...
-```
\ No newline at end of file
+# backend

```

Comment:
What is the goal, to make it convenient to start the whole project in one command and not to install anything, you can leave dbeaver.